src/charm.py

```diff
@@ -50,6 +51,12 @@ def __init__(self, *args):
             self.framework.observe(self.prometheus_provider.on.targets_changed,
                                    self._on_scrape_targets_changed)
 
+        self.alertmanager_lib = AlertmanagerConsumer(self,
+                                                     relation_name="alertmanager",
+                                                     consumes={'alertmanager': '>0.0.0'})
```

Comment:
If I remember correctly you can also pass None for version if there is no restriction on version. Please do check this if you want to do it that way. Also I presume you have indeed ascertained that any Prometheus version will work with any version of Alertmanager.

Reply:
IIRC, `None` was producing a WARNING in the log. Regardless, it's good to have it versioned.
After we merge the alertmanager PR, I can change this to correspond.

Reply:
Thanks. I will look at that warning and think about it. Do paste that warning if you will. And if you care try empty string too.

Reply:
changed